job_runner.py

```diff
@@ -0,0 +1,97 @@
+import os
+from jsonschema import validate, ValidationError
+import yaml
+from test_runner import TestRunner
+from compliance_suite.exceptions.ComplianceException import ComplianceException
+from compliance_suite.functions.colored_console import print_blue, print_red, print_green, print_yellow
+
+
+class JobRunner():
+
+    def __init__(self, tags):
+        self.path = os.getcwd()
+        self.tags = tags
+        self.test_count = 0
```

Comment:
Could also be obtained from `.test_status` (see next comment)

Reply:
We will require this variable to print which test no. is being run in the logs and update the test status.

Reply:
Okay, fair enough